lua/rose-pine.lua

```diff
@@ -941,6 +941,22 @@ local function set_highlights()
 		RenderMarkdownTableHead = { fg = palette.subtle },
 		RenderMarkdownTableRow = { fg = palette.subtle },
 		RenderMarkdownUnchecked = { fg = palette.subtle },
+
+		-- MagicDuck/grug-far.nvim
+		GrugFarHelpHeader = { fg = palette.pine },
+		GrugFarHelpHeaderKey = { fg = palette.gold },
+		GrugFarHelpWinActionKey = { fg = palette.gold },
+		GrugFarHelpWinActionPrefix = { fg = palette.foam },
+		GrugFarHelpWinActionText = { fg = palette.pine },
+		GrugFarHelpWinHeader = { link = "FloatTitle" },
+		GrugFarInputLabel = { fg = palette.foam },
+		GrugFarInputPlaceholder = { link = "Comment" },
+		GrugFarResultsActionMessage = { fg = palette.foam },
+		GrugFarResultsChangeIndicator = { fg = groups.git_change },
+		GrugFarResultsHeader = { fg = palette.pine },
+		GrugFarResultsMatch = { link = "Search" },
```

Comment:
If we want the grug matches to be more prominent we can link to the "CurSearch" instead

```suggestion
		GrugFarResultsMatch = { link = "CurSearch" },
```

Reply:
That works.